servers/src/main/java/tachyon/worker/block/TieredBlockStore.java

```diff
@@ -89,17 +89,19 @@ public TieredBlockStore(TachyonConf tachyonConf) throws IOException {
     mTachyonConf = Preconditions.checkNotNull(tachyonConf);
     mMetaManager = BlockMetadataManager.newBlockMetadataManager(mTachyonConf);
     mLockManager = new BlockLockManager(mMetaManager);
+    BlockMetadataManagerView initManagerView =
+        new BlockMetadataManagerView(mMetaManager, new HashSet<Integer>() , new HashSet<Long>());
```

Comment:
You may want to create one view for the allocator and one to the evictor just for isolation